torch/nn/modules/rnn.py

```diff
@@ -25,6 +26,17 @@ def __init__(self, mode, input_size, hidden_size,
         self.bidirectional = bidirectional
         num_directions = 2 if bidirectional else 1
 
+        if not isinstance(dropout, numbers.Number) or not 0 <= dropout <= 1 or \
+                isinstance(dropout, bool):
```

Comment:
lol, I didn't know bool is an instance of Number

Reply:
in fact `bool`s are instances of `int`!